SConstruct

```diff
@@ -592,6 +592,9 @@ if selected_platform in platform_list:
                 "C++17. Supported versions are Clang 6 and later."
             )
             Exit(255)
+        if env["platform"] == "ios" and env["arch"] == "x86_64" and not env["ios_simulator"]:
```

Comment:
This check probably should be it the platform/ios/detect.py instead of here.

Reply:
Moved! Thank you.